blog/models.py

```diff
@@ -215,21 +215,19 @@ class BlogPage(MetadataPageMixin, MediaPageMixin, Page):
 
     link_to_original_post = models.URLField(blank=True)
 
-    lead_image = models.ForeignKey(
-        'common.CustomImage',
-        null=True,
-        blank=True,
-        on_delete=models.SET_NULL,
-        related_name='+',
-    )
+    lead_graphic = StreamField([
+        ('image', ImageChooserBlock()),
+        ('vertical_bar_chart', VerticalBarChart()),
+        ('tree_map_chart', TreeMapChart()),
+        ('bubble_map_chart', BubbleMapChart()),
+    ], use_json_field=True, blank=True, default=[], max_num=1)
 
-    teaser_image = models.ForeignKey(
-        'common.CustomImage',
-        null=True,
-        blank=True,
-        on_delete=models.SET_NULL,
-        related_name='+',
-    )
+    teaser_graphic = StreamField([
```

Comment:
TODO: double check all of the areas that use `teaser_image` to make sure they all get migrated over and don't break.